### ISSUE TITLE
Import GraphQL fragments.

### ISSUE THREAD

**Issue by envygeeks** [gridsome/gridsome#377]
## Summary

When it comes to things like posts, authors, pages, and other various pieces that might share a common structure, it would be nice to be able to import a set of GraphQL fragments that hold common queryable fields, that gets parsed into GraphQL early.

### Basic example

`src/fragments/post.graphql`
```graphql
fragment PostParts on post {
  title
  path

  author {
    ...AuthorParts
  }
}
```

### Motivation

Because of the way posts are handled in Gridsome it becomes necessary to share infrastructure beyond the current component, and start making things more manageable.

**Comment by envygeeks** [comment]
If this isn't possible, or not wanted, can the GraphQL layer be exposed at least? I can do it myself.

**Comment by envygeeks** [comment]
Closed for #232
